.github/scripts/build.sh

```diff
@@ -44,7 +44,7 @@ for DIST in ${DEB_DIST}; do
 
     export DESCRIPTION='man1'
     export ORIGIN='https://github.com/ansible/ansible'
-    pip install --requirement requirements.txt docutils
+    pip install --break-system-packages --requirement requirements.txt docutils
```

Comment:
Can we document why this option has been added

Reply:
this has been reverted by switching to using a venv instead